mobile/examples/React_Native/ort-rn-basic-usage/App.tsx

```diff
@@ -0,0 +1,69 @@
+import { StatusBar } from 'expo-status-bar';
+import { Alert, Button, StyleSheet, Text, View } from 'react-native';
+
+import * as ort from 'onnxruntime-react-native';
+import { Asset } from 'expo-asset';
+
+let myModel: ort.InferenceSession;
+
+async function loadModel() {
+  try {
+    // TODO: Can also add example code to demonstrate loading model from bytes here
+    const assets = await Asset.loadAsync(require('./assets/mnist.ort'));
```

Comment:
Now that we're using the full package we should use a .onnx file for simplicity. One less step for the user to do. 

Could also tell the user they can view the model in Netron to see that there's one input and one output, and the data types and shapes of those.